ecs-cli/modules/cli/cluster/cluster_app.go

```diff
@@ -348,6 +366,19 @@ func createCluster(context *cli.Context, awsClients *AWSClients, commandConfig *
 	return cfnClient.WaitUntilCreateComplete(stackName)
 }
 
+func getContainerInstanceARNSetting(client ecsclient.ECSClient) (*ecs.Setting, error) {
```

Comment:
See above comment, but I might consider renaming this to reflect the fact that it's used to determine if tagging is possible on container instances. Would also be worth adding a comment since this is probably not obvious to newcomers.

Reply:
Apropos, i refer you to your future self ;) : https://github.com/aws/amazon-ecs-cli/pull/731/files#diff-499d2f9fc374c259c658f974cbc87b46R564

Reply:
😛